src/Native/Unix/System.Native/pal_random.c

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
since you are touching this file, could you also fix the typo on the second last line https://github.com/dotnet/corefx/blob/1f59cb91f9350732c747e6d60422c9c579e1dfed/src/Native/Unix/System.Native/pal_random.cpp#L101 to `#endif // HAVE_`